### PR TITLE
2022.0.0.431

### PR DESCRIPTION
**Pull request by chbwien**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57174)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please update your CI/CD pipelines with multi-file manifests.

**Comment by chbwien** [comment]
What is wrong with a singleton? It's the descibed way? Unfortunately not the full 1.0.0 singleton is ok for the approval (e.g. channel is included in the singleton 1.0.0 but is not accepted).

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> What is wrong with a singleton? It's the descibed way? Unfortunately not the full 1.0.0 singleton is ok for the approval (e.g. channel is included in the singleton 1.0.0 but is not accepted).

"Note: The singleton manifest format has been deprecated in the Windows Package Manager Community Repository. The Windows Package Manager 1.2 client still supports singleton manifests." in https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.1.0/singleton.md#minimal-singleton-yaml-file-example

All packages are being converted to multi-file manifests now.

File: manifests/t/TUitTUWien/TUtoolbox/2022.0.0.431/TUitTUWien.TUtoolbox.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PackageVersion: 2022.0.0.431
+PackageLocale: de-DE
+Publisher: TU Wien - TU.it
+PublisherUrl: https://www.it.tuwien.ac.at/
+PublisherSupportUrl: https://www.it.tuwien.ac.at/services/beratung-und-servicedesk/servicedesk/service-center
+# PrivacyUrl: 
+Author: Christian Beck
+PackageName: TUtoolbox
+PackageUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox
+License: © 2021-2022 TU.it, TU Wien.
```
Comment (Trenly) [inline manifests/t/TUitTUWien/TUtoolbox/2022.0.0.431/TUitTUWien.TUtoolbox.locale.de-DE.yaml:13]: A copyright is not a license; Please update this to reflect the package terms as `Proprietary`, `MIT`, `GPL-2.0`, or whatever the actual software License is

File: manifests/t/TUitTUWien/TUtoolbox/2022.0.0.431/TUitTUWien.TUtoolbox.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PackageVersion: 2022.0.0.431
+PackageLocale: en-US
+Publisher: TU Wien - TU.it
+PublisherUrl: https://www.it.tuwien.ac.at/
+PublisherSupportUrl: https://www.it.tuwien.ac.at/services/beratung-und-servicedesk/servicedesk/service-center
+# PrivacyUrl: 
+Author: Christian Beck
+PackageName: TUtoolbox
+PackageUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox
+License: © 2021-2022 TU.it, TU Wien.
```
Comment (Trenly) [inline manifests/t/TUitTUWien/TUtoolbox/2022.0.0.431/TUitTUWien.TUtoolbox.locale.en-US.yaml:13]: A copyright is not a license; Please update this to reflect the package terms as `Proprietary`, `MIT`, `GPL-2.0`, or whatever the actual software License is

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/t/TUitTUWien/TUtoolbox/2022.0.0.431/TUitTUWien.TUtoolbox.locale.en-US.yaml (hunk 2 above)
Comment (chbwien) [reply manifests/t/TUitTUWien/TUtoolbox/2022.0.0.431/TUitTUWien.TUtoolbox.locale.en-US.yaml]: I changed it, but a lot of other packages show the same. And this information was ok for the last submits.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**